dice.js

```diff
@@ -0,0 +1,18 @@
+// Homework #3
+
+/// Part 2
+const rollDice = function() {
+  const num1 = Math.floor(Math.random() * 6) + 1;
+  const num2 = Math.floor(Math.random() * 6) + 1;
+  const total = num1 + num2;
+
+  console.log("Dice rolled are " + num1 + " and " + num2 + ". Sum is " + total);
+};
+
+rollDice();
+
+const rollDice = function() {
```

Comment:
this function is not needed, never have to functions with the same name in the same file.